wallet-new/test/Spec.hs

```diff
@@ -17,3 +26,14 @@ main = hspec $ do
     API.spec
     Swagger.spec
     ReqSpec.spec
+
+    eqProps @WalletAddress
+    eqProps @Address
+    eqProps @Wallet
+    eqProps @Transaction
```

Comment:
I'm getting some weird results with `elem` and `shouldBe`. When I don't trust things, I write tests.